Azure/azure-cli
# {sql} Test fix for Managed Instance Start Stop

**Related command**


There are no new commands added, this is purely a test fix.

**Description**


This PR is a fix for a leftover item from https://github.com/Azure/azure-cli/pull/26979. It includes a fix for the test to not use existing resources but instead to create and delete a new resource to perform the test on.

**Testing Guide**
No new commands added.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
